app/src/components/pages/errorPage/errorPage.tsx

```diff
@@ -29,13 +29,11 @@ export default function ErrorPage() {
         height="68"
         style={{ marginBottom: "48px" }}
       />
-      <Heading overline="Error 500" level="h3">
-        Something went wrong on our end.
-      </Heading>
+      <Text size="overline1">ERROR 500</Text>
+      <Heading level="h3">Something went wrong on our end.</Heading>
```

Comment:
Was this a vqa feedback?

Reply:
Yes– spacing on header not what was there either bc of VDL mismatch or Lily intervention

Reply:
Probably VDL mismatch and if you have to break this out into two components, let's bring this to the DS standup tomorrow (10/17)